### PR TITLE
Fix object trailing commas that last property is ignored

### LINKED ISSUE #5678: Comma Dangle error if prettier ignore is set to ignore last property in object
**Prettier 1.15.3**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEAnOAzXNACwBUIBrBABQEMBPAGwhoBNMBeTYAHSk38ww6ABzhJMAclwE4xMpSgSANLwGY0AVwBG4mEQCWaAHQB9fSxV8B2CAzGCDxm3cv9uMAPQfMw3DBj6cNgAtPoA5lAQuKoCcAAewuKRAO6YANSYABQAbAAMmABUmHmFmABMACylAJy5AJRKmF5ZtZgs9Gh1MQC+ANwgSiAQwgHoyKA02DbJtLhQaMggNABuEOYDIFrYNGCUMADKwjv6UGHIMNgacINEMAC2DADqBvBoR2Bw+who+gHLv3RFmA0AtBic0EEYFRtmE7jRkHgaAwIYMAFZoOIAIW2uzgBxodzgABkTnAEUiUSB0XF9icwnYAIoaCDwcnI64gI7YCHYRZaGhaRjQDa+E4wR7mPTIAAcuUGvggEMe22Ei18siCyzJg1wAEcNPpcNCaLD4UhEezBhC7vpzpcOT9TozmazzRSOTABRKWFKkGVBhcaPoGHSAMIQO5wxaRKDakAaCEkAULN3s7rdIA)
```sh
--parser babylon
```

**Input:**
```jsx
const refreshTokenPayload = {
    type: 'refreshToken',
    sub: this._id,
    role: this.role,
  	// prettier-ignore
    exp: now + (60 * 60 * 24 * 90), // (90 days)
  };
```

**Output:**
```jsx
const refreshTokenPayload = {
  type: "refreshToken",
  sub: this._id,
  role: this.role,
  // prettier-ignore
  exp: now + (60 * 60 * 24 * 90)
};

```

**Expected behavior:**
```jsx
const refreshTokenPayload = {
  type: "refreshToken",
  sub: this._id,
  role: this.role,
  // prettier-ignore
  exp: now + (60 * 60 * 24 * 90), // <-- prettier removes the comma and then the linter blows up
};

```

FYI, I used prettier ignore so I could keep my parentheses https://github.com/prettier/prettier/issues/187

### PR DESCRIPTION
**Pull request by fisker**
Fixes #5678

- [x] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [x] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by fisker** [comment]
I guess you have to add `prettier-ignore` to object. :)

**Comment by fisker** [comment]
[Array use the same logic ](https://deploy-preview-8111--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEBDABAXnQbQDpTroD0x6ADgE5wwwCWclAtHQOZQTUFEBMBAugG4CBDNnyESZKjXqMW7TnG7oeAGgEiopCtVoNmbDlyhjcKvlH4g1ICOXrQAzslCpKlCAHcACu4QuKKgANl6oAJ4utgBGlKhgANY0AMrk8XRQrMgwlACucLZwALbRcAAmZeUAMqiZuaiscABinEWo+pnIaLkwEDYgABYwRcEA6gN08E5pYHDJAZN0AG6T4V1gTlEgGU6MMD5xrG3IAGYhu7YAVk4AHgBCcYkpqEVwVRlwp+cFINc3yRlWME4ABFXIQeBfYIXEBpSi7ShdaKoUrBfpUDIwUZ0MowAbIAAcAAZbFQILtRnFyF0ZAilp9bABHcHwA72QJoJxMKBwcrlfrUZl0agHBrHJBnaE-XZFOjZPLSwHAsEQz4S762GAo7G4-FIdQgHKoOjBQEAYQgRXFIDgTgArP1crsACoowKSmFLfIASSglVgyTAlDoDgAgn7kjBwsCobsAL5xoA)

File: tests/object_property_ignore/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -0,0 +1,665 @@
...
+foo = {
+  _: "",
+  // prettier-ignore
+  bar:            1, // comment
+};
+
+foo = {
+  _: "",
+  // prettier-ignore
+  bar:            1 /* comment */,
```
Comment (fisker) [inline tests/object_property_ignore/__snapshots__/jsfmt.spec.js.snap:79]: This comment is move from after `,` to before `,`. [Array do the same](https://deploy-preview-8111--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEBDABAXnQbQDpTroD0x6ADgE5wwwCWclAtHQOZQTUFEBMANEUFCixAFTpIAW0kIY6UcQIBdEHxARy9aAGdkoVJUoQA7gAUDCXSlQAbY6gCeutQCNKqMAGsaAZXIe6KFZkGEoAVzg1OEkXOAATOPiAGVQgsNRWOAAxTklUWkDg6zCYCFUQAAsYSRsAdQq6eG1-MDgfS0a6ADdGh2RwbWcQQO1GGFN3VjzkADNbUbUAK20ADwAhdy9fVBkkwLhZ+ciQZZWfQps4AEUwiHhDmwWQf0pRyn6XVFibcqpAmFqdDiMAqyAAHAAGNRUCCjWrucj9KhwN5dA5qACOt3gEw0VjQ2iYUDg8Xi5WoWLo1AmGWmSDmj2Oo0kdBC4SZF2u2IO9KOahgX0BwNBSH4IFCqDoNkKAGEINJUP0UQBWcphUYAFS+VgZTy6EQAklBErAfGBKHRNABBY0+GAOS4PUYAX2dQA)

File: tests/object_property_ignore/ignore.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  bar:            1,         /* comment */
+}
+
+foo = {
+  _: '',
+  // prettier-ignore
+  bar:            /* comment */          1,
+}
+
+/* RestElement */
```
Comment (thorn0) [inline tests/object_property_ignore/ignore.js:31]: This is `SpreadElement`. Rest is destructuring.

**Comment by fisker** [comment]
I was going to fix that comment...

**Comment by evilebottnawi** [comment]
@fisker oh, sorry, feel free to send a new PR

**Pull request merged**